src/scripts.js

```diff
@@ -391,12 +220,12 @@ function findCheckedPantryBoxes() {
   let selectedIngredients = pantryCheckboxInfo.filter(box => {
     return box.checked;
   })
-  showAllRecipes();
+  domUpdates.showAllRecipes(recipeData);
   if (selectedIngredients.length > 0) {
     findRecipesWithCheckedIngredients(selectedIngredients);
   }
 }
-
+//We can improve the function below if we make it look for recipes that include only one checked item, so we can get multiple recipes that call for a specific ingredient.
```

Comment:
I'd love to see us make this function better. Currently it doesn't seem to be working in a very efficient way. It does what we want it to do, but it could definitely be improved